Corona-Warn-App/src/main/res/layout/fragment_uqs_info.xml

```diff
@@ -1,12 +1,211 @@
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginEnd="26dp"
+                    android:text="@string/qr_code_info_screen_fourth_bulletpoint_text"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toEndOf="@id/first_bulletpoint_icon"
+                    app:layout_constraintTop_toTopOf="@id/fourth_bulletpoint_icon" />
+
+                <TextView
+                    android:id="@+id/data_details"
+                    style="@style/subtitleMedium"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="40dp"
+                    android:text="@string/qr_code_info_screen_data_details_text"
+                    app:layout_constraintEnd_toEndOf="@id/fourth_bulletpoint_text"
+                    app:layout_constraintStart_toStartOf="@id/fourth_bulletpoint_icon"
+                    app:layout_constraintTop_toBottomOf="@id/fourth_bulletpoint_text" />
+
+                <TextView
+                    android:id="@+id/booking_provider_details"
+                    style="@style/subtitleMedium"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/qr_code_info_screen_booking_provider_text"
+                    app:layout_constraintEnd_toEndOf="@id/data_details"
+                    app:layout_constraintStart_toStartOf="@id/data_details"
+                    app:layout_constraintTop_toBottomOf="@id/data_details" />
+
+                <de.rki.coronawarnapp.ui.view.MoreInformationView
+                    android:id="@+id/privacy_information"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/spacing_medium"
+                    android:layout_marginTop="@dimen/spacing_normal"
+                    android:layout_marginEnd="@dimen/spacing_medium"
+                    android:layout_marginBottom="@dimen/spacing_small"
+                    android:focusable="true"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/booking_provider_details"
+                    app:titleText="@string/qr_code_info_screen_privacy_policy" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </androidx.core.widget.NestedScrollView>
+
+    </androidx.coordinatorlayout.widget.CoordinatorLayout>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Dark mode has different icon color , see Test icon
<img width="440" alt="Screenshot 2021-11-17 at 12 25 41" src="https://user-images.githubusercontent.com/25054729/142192004-99bfb5ff-9f16-402d-b15e-2fa236c30d23.png">

Reply:
Updated the colors.